style.css

```diff
@@ -0,0 +1,94 @@
+body {
+  display: flex;
+  flex-direction: column;
+  height: 480px;
+  width: 320px;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  height: 74px;
+}
+
+.menu {
+  height: fit-content;
+  padding: 0;
+  margin-top: 19px;
+  margin-right: 16px;
+  background-color: transparent;
+  background-repeat: no-repeat;
+  border: none;
+  cursor: pointer;
+  overflow: hidden;
+  outline: none;
+}
+
+section {
+  display: flex;
+  flex-direction: column;
+  margin-left: 16px;
+  margin-right: 16px;
+  margin-top: 30px;
+  margin-bottom: 16px;
+}
+
+h1 {
+  font-family: "Playfair Display", serif;
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 50px;
+  margin-top: 0;
+  margin-right: 68px;
+  margin-bottom: 0;
+}
+
+p {
+  margin-top: 20px;
+  margin-bottom: 40px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+}
+
+.btn {
+  display: flex;
+  justify-content: center;
+  background-color: #1a2236;
+  color: white;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+}
```

Comment:
Awesome job creating a button with the correct effects 😄! For a perfect representation of the Figma button, kindly increase the button's padding and remove the border:

| Figma design  | Your design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/42799579/198755407-bb924e47-b047-4596-9274-7e7b1cddc126.png)  | ![image](https://user-images.githubusercontent.com/42799579/198755399-244c8e94-4d62-4ea2-8ac0-a8ef16c8606c.png)  |

> Keep in mind that this project's goal is to improve your ability to follow design guidelines (imagine Microverse as a client requesting a website) 😉.